charts/harvester-cloud-provider/values.yaml

```diff
@@ -6,9 +6,9 @@ replicasCount: 1
 
 image:
   repository: rancher/harvester-cloud-provider
-  pullPolicy: Always
```

Comment:
`Always` is OK for `master-head` verison.

Reply:
Now CSI is `IfNotPresent`, modify to `Always` too?

Reply:
Now CSI is `IfNotPresent`, modify to `Always` too?

Reply:
Yes, it makes sense.